.changeset/rude-lizards-scream.md

```diff
@@ -0,0 +1,51 @@
+---
+'astro': minor
+---
+
+Experimental support for i18n routing. 
+
+Astro's experimental i18n routing allows you to add your multilingual content with support for configuring a default language and URL path convention, computing relative page URLs, and accepting preferred languages provided by your visitor's browser browser.
```

Comment:
```suggestion
Astro’s experimental i18n routing API allows you to add your multilingual content with support for configuring a default language, computing relative page URLs, and accepting preferred languages provided by your visitor’s browser. You can also specify fallback languages on a per-language basis so that your visitors can always be directed to existing content on your site.
```
I realized we didn't mention fallback, which is a cool thing!